src/store/location-redux.ts

```diff
@@ -0,0 +1,71 @@
+export interface LocationState {
+  currentLocation: string;
+  prevSearches: string[];
+}
+export enum LocationActions {
+  SAVE_CURRENT_LOCATION,
+  SAVE_SEARCH
+}
+export interface LocationActionConfig {
+  type: LocationActions;
+  payload: string;
+}
+
+const getPrevSearches = (): string[] => {
+  const prevSearches: string | null = localStorage.getItem('prevSearches');
+  if (prevSearches) {
+    return JSON.parse(localStorage.getItem('prevSearches')!);
+  } else return [];
```

Comment:
could be used ternary operator instead. or at least please use the same syntax for if and else conditions

Reply:
ok, changed to ternary operator